tests/integration/records/multi_db/test_records_table2table.py

```diff
@@ -105,8 +105,8 @@ def source2target(self):
 if __name__ == '__main__':
     set_stream_logging()
 
-    for source in DB_TYPES:
-        for target in DB_TYPES:
+    for source in ['redshift']:  # DB_TYPES: TODO
+        for target in ['bigquery']:  #  DB_TYPES: TODO
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker